src/model/Localization.cxx

```diff
@@ -0,0 +1,27 @@
+#include "Localization.hxx"
+#include "Filesystem.hxx"
+#include "../util/LOG.hxx"
+
+using namespace std;
```

Comment:
Do not open namespaces. Instead include exactly what you need to prevent conflicts
```suggestion
using std::string;
```

Reply:
I think it is quite safe in .cxx files, stays locally. But will change as you say.